client/starwhale/cli/assistance/cli.py

```diff
@@ -0,0 +1,46 @@
+from typing import Tuple
+
+import click
+
+from .host import CommandRetriever
+from .remote import CommandRunner
+
+
+@click.group("assistance", help="Remote assistance")
```

Comment:
```python
@click.group("assistance", help="Remote assistance")
```
-->
```python
from starwhale.utils.cli import AliasedGroup
@click.group("assistance", cls=AliasedGroup, help="Remote assistance")
```
use `AliasedGroup`, the users can use short alias command for `assistance`, for example: `swcli ass`, `swcli assist` are both ok for `assistance`.